src/app/components/publiclists/teamlist/teamlist.component.html

```diff
@@ -1,91 +1,63 @@
-                </div>
-                <div *ngIf="value.team_url == ''" class="radio-button-text pointer pointer content">
-                  <strong class="text-med-black fw-semibold">Team: </strong>{{value.team_name}}
-                  <br><strong class="text-med-black fw-semibold">Created By: </strong>{{value.created_by}}
-                </div>
-                <div class="clearfix"></div>
-              </li>
-
-              <div class="pagination" *ngIf="paginationDetails.showPagination">
-                <div class="row row-lr-margin">
-                  <div class="col-md-6 col-sm-12 left-align col-lr-pad">
-                    <button [ngClass]="paginationDetails.isPrev" class="btn-floating btn-pagination waves-effect waves-light " 
-                    (click)="loadPaginationData(previous)">
-                      <i class="fa fa-chevron-left"></i>
-                    </button>
-                    <span class="pagination-title">
-                      <strong class="text-med-black fw-semibold content">
-                        Page {{paginationDetails.currentPage}} of {{paginationDetails.totalPage}}
-                      </strong></span>
-                    <button [ngClass]="paginationDetails.isNext" class="btn-floating btn-pagination waves-effect waves-light" 
-                    (click)="loadPaginationData(next)">
-                      <i class="fa fa-chevron-right"></i>
-                    </button>
-                  </div>
-                  <div class="col-md-6 col-sm-12 col-lr-pad">
-                    <div class="create-challenge align-right" *ngIf="isOnChallengePage">
-                      <button [disabled]="!selectedTeam" class="btn btn-waves-effect grad-rec-btn waves-dark ev-btn-dark 
-                        grad-btn-dark fw-light fs-14" (click)="participateInChallenge()">
-                        Continue
-                      </button>
-                    </div>
-                  </div>
-                </div>
+<section class="ev-sm-container ev-view ">
+  <div class="row">
+    <div class="col s12 m6">
+      <div class=" ev-card-panel ev-z-depth-5">
+        <div class="ev-md-container ev-panel-title"><strong>{{teamSelectTitle}}</strong></div>
+        <div class="ev-md-container ev-card-body exist-team-card">
+          <div *ngIf="allTeams.length === 0" class="page-msg">No team exists for now. Start by creating a new team!</div>
+          <ul class="collapsible" data-collapsible="accordion">
+            <app-teamcard *ngFor="let team of filteredTeams"
+                          [team]="team" [selected]="team['isSelected']" [isOnChallengePage]="isOnChallengePage"
+                          (deleteTeamCard)="deleteTeamWrapper()($event)" (editTeamCard)="editTeamWrapper()($event)" (addMembersTeamCard)="addMembersToTeamWrapper()($event)"
+            (selectTeamCard)="selectTeamWrapper()($event)">
+            </app-teamcard>
+          </ul>
```

Comment:
When I go to say, 2nd page in pagination i.e. **(Page 2 of 2)**, all teams from the previous pages are also displayed with the remaining teams of this page.

Reply:
okay, correcting it

Reply:
done